ui/v2.5/src/locales/en-GB.json

```diff
@@ -193,7 +193,9 @@
         "password_desc": "Password to access Stash. Leave blank to disable user authentication",
         "stash-box_integration": "Stash-box integration",
         "username": "Username",
-        "username_desc": "Username to access Stash. Leave blank to disable user authentication"
+        "username_desc": "Username to access Stash. Leave blank to disable user authentication",
+        "trusted_proxies": "Trusted proxies",
```

Comment:
I believe the intention is to sort these by on the keys. So it should be above:
```
"stash-box_integration": ...,
"trusted_proxies": ...,
"trusted_proxies_desc": ...,
"username": ...,
```

Reply:
Reordered these